### PR TITLE
Adding a tracking issue template

### PR DESCRIPTION
**Pull request by qdequele**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/tracking-issue.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+- What is changing for end-users.
+- How it works.
+- What is breaking?
+- Examples.
+
+# Implementation
+
+Explain the technical specificities that will need to be known or done in order to implement this proposal.
+
```
Comment (Kerollmops) [inline .github/ISSUE_TEMPLATE/tracking-issue.md:30]: Could you add a "Documentation draft" header which must be a short documentatio that user will probably see in the documentation, a short explanation on how to use the feature?

Comment (qdequele) [reply .github/ISSUE_TEMPLATE/tracking-issue.md:30]: It is already here https://github.com/meilisearch/MeiliSearch/pull/785/files#diff-cdcc247fa1ab0b29514a3e0c4ad2248dR18-R20 
It is not enough?

**Comment by qdequele** [comment]
I think it could be merge?!

**Comment by MarinPostma** [comment]
Yes sure!
Bors r+

**Pull request merged**